packages/typespec-ts/src/modular/serialization/buildSerializerFunction.ts

```diff
@@ -0,0 +1,214 @@
+import { toCamelCase, toPascalCase } from "../../utils/casingUtils.js";
+import { Type as ModularType } from "../modularCodeModel.js";
+import {
+  getPropertySerializationPrefix,
+  getRequestModelMapping
+} from "../helpers/operationHelpers.js";
```

Comment:
Is the derializers in preview scope?

Reply:
It is something we'll have to iterate during preview. As far as my exploration went, iterating on this won't impact the API